lib/handlers/allow.js

```diff
@@ -68,6 +69,21 @@ function allow (mode) {
         }
       }
     }
+
+    // check user is owner. Find owner from /meta
+    let graph
+    try {
+      // do not check for permission. Owner has all rights.
+      // TODO check permissions on .meta ???
+      graph = await ldp.getGraph(rootUrl + '/.meta')
+      const SOLID = $rdf.Namespace('http://www.w3.org/ns/solid/terms#')
+      const owner = await graph.any(null, SOLID('account'), $rdf.sym(rootUrl + '/'))
```

Comment:
Not sure about the role that `solid:account` plays towards the role that `solid:owner` plays. `solid:account` typically refers to the root container / Storage

Reply:
`solid:account` seems to be what NSS is already using for this.  So this fix can piggyb ack on that.